### PR TITLE
1password doc updated

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — COMMENTED** [review]
@JV0812 , I tried running this locally and got the following error:

```
Compiled with problems:
×
ERROR in ./docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md 104:0-77
Module not found: Error: Can't resolve 'raw-loader' in '/Users/kpohas/sumologic-documentation/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework'
ERROR in ./docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md 105:0-74
Module not found: Error: Can't resolve 'raw-loader' in '/Users/kpohas/sumologic-documentation/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework'
```

**Comment by JV0812** [comment]
Can you please install the raw loaded component, using `yarn add raw loaded`

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 1]
```diff
@@ -2,359 +2,122 @@
...
-
-## Rules
+## Data collected
 
-* JSON is the only supported log format.
-* Data is collected in five minute intervals.
+| Polling Interval | Data |
+| :--- | :--- |
+| 5 min |  [Sign-in Attempts](https://support.1password.com/events-api-reference/#sign-in-attempts)
+| 5 min |  [Item Usage](https://support.1password.com/events-api-reference/#item-usage)
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:26]:
```suggestion
| 5 min |  [Sign-in Attempts](https://developer.1password.com/docs/events-api/reference/#post-apiv1signinattempts)
| 5 min |  [Item Usage](https://developer.1password.com/docs/events-api/reference/#post-apiv1itemusages)
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 2]
```diff
@@ -2,359 +2,122 @@
...
+| 5 min |  [Audit Events](https://developer.1password.com/docs/events-api/reference/#post-apiv1auditevents)
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:27]:
```suggestion
| 5 min |  [Audit events](https://developer.1password.com/docs/events-api/reference/#post-apiv1auditevents)
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 3]
```diff
@@ -2,359 +2,122 @@
...
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/img/c2c/1password/example.json';
+import MyComponentSource from '!!raw-loader!/img/c2c/1password/example.json';
+import TerraformExample from '!!raw-loader!/img/c2c/1password/example.tf';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<img src={useBaseUrl('img/integrations/1password/1password.png')} alt="Thumbnail icon" width="45"/>
+<img src="https://app_icons.s3.amazonaws.com/1Password.png" alt="Thumbnail icon" width="80"/>
 
 The 1Password Source provides a secure endpoint to receive Sign-in Attempts, Item Usage, and Audit Events from the [1Password Event API](https://support.1password.com/events-api-reference/). It securely stores the required authentication, scheduling, and state tracking information.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:19]:
```suggestion
The 1Password Source provides a secure endpoint to receive sign-in attempts, item usage, and audit events from the [1Password Event API](https://support.1password.com/events-api-reference/). It securely stores the required authentication, scheduling, and state tracking information.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 4]
```diff
@@ -2,359 +2,122 @@
...
-   <td>The Source will retry for up to 90 minutes, after which it quits.
-   </td>
-   <td>FirstPartyGenericError
-   </td>
-  </tr>
-</table>
+## JSON schema
 
-### Restarting your Source
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for more details.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:75]:
```suggestion
Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [How to Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for more details. 
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md [hunk 5]
```diff
@@ -2,359 +2,122 @@
...
 
 ![troubleshooting.jpg](/img/send-data/1password-troubleshooting.jpg)
 
-**Error Code**: `401` <br />
-**Error Details**: `{"Error":{"Message":"Unauthorized"}}`
-
 To resolve these errors:
 - Make sure the Base URL matches your domain.
 - Make sure correct API Token is used to configure the source.
 - If you're still seeing the `401 Unauthorized error` in the **Status** column, regenerate the API Token by following [these Authentication steps](#authentication) and then updating the API Token for the source.
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md:117]:
```suggestion
- If you're still seeing the `401 Unauthorized error` in the **Status** column, regenerate the API Token by following [these Authentication steps](#vendor-configuration) and then updating the API Token for the source.
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
I have minor suggestions, @JV0812.

**Comment by jpipkin1** [comment]
@kimsauce, are we good with this PR updating the `package.json` and `yarn.lock` files?

**Comment by kimsauce** [comment]
> @kimsauce, are we good with this PR updating the `package.json` and `yarn.lock` files (resulting from running `yarn add raw-loader`)?

Just ran it locally and worked for me! Yes sounds good.

**Comment by JV0812** [comment]
Thanks for your review @jpipkin1. I have fixed your comments.

**Review by jpipkin1 — APPROVED** [review]

**Pull request merged**